llvm/test/CodeGen/AMDGPU/GlobalISel/non-entry-alloca.ll

```diff
@@ -75,8 +75,8 @@ bb.2:
   store volatile i32 0, ptr addrspace(1) undef
   ret void
 }
-; DEFAULTSIZE: .amdhsa_private_segment_fixed_size 4112
-; DEFAULTSIZE: ; ScratchSize: 4112
+; DEFAULTSIZE: .amdhsa_private_segment_fixed_size 16
```

Comment:
This seems a bit suspect. It used to be about 4k and is now 16. Are we out by a factor of 1024 somewhere?

Reply:
My understanding is that it's supposed to be `0` if the backend could not statically determine it, so it's unlikely to be due to that change in COV5. Maybe it's not matching the full line of 16384 or similar?